test/ibm/runtime/test_runtime.py

```diff
@@ -637,12 +659,14 @@ def _upload_program(self, name=None, max_execution_time=300,
         """Upload a new program."""
         name = name or uuid.uuid4().hex
         data = "def main() {}"
+        metadata = self.DEFAULT_METADATA
```

Comment:
Just so one bad test doesn't impact the next, can you make a copy of the `DEFAULT_METADATA` instead of updating it in place?